packages/plugins/type-designer/src/public/type-designer.simple.scd

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<SCL version="2007" revision="B" release="4" xmlns:td="http://localhost:53515/type-designer.schema.xsd">
+<Header id="Type Designer Plugin simple test file"/>
+<DataTypeTemplates>
+	<Private name="Type designer Plugin">
+		<td:SubstationTypesRoot>
```

Comment:
This was meant to facilitate our XML CRUD operations : with a scoped field per type. It correspond to each columns that we will have in the type designer plugin. You might think it's an overkill feature for our needs, and that it reduce the readability of the XML file. If so, I will remove this level in both the schema and the dummy file.

Reply:
The querries will be very similar

1. `DataTypeTemplates > Private > SubstationTypesRoot > SubstationType`
2. `DataTypeTemplates > Private > SubstationType`

When creating a new type in 1. you would have to check if the `SubstationTypesRoot` exists or create it otherwise, while for 2. you would have to consider where to insert it, so the `Types` are aligned.

Personally I am leaning towards leaving out the Root element, but I think it is a valid way of doing things